src/components/pages/pages.spec.js

```diff
@@ -2,13 +2,8 @@ import React from 'react';
 import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup';
 import { shallow, mount } from 'enzyme';
 import BasePages, { Page } from './pages.component';
-import Heading from '../heading/heading';
 import { rootTagTest } from '../../utils/helpers/tags/tags-specs/tags-specs';
-import {
-  PagesContent
-} from './pages.style';
 import classicTheme from '../../style/themes/classic';
-import { assertStyleMatch } from '../../__spec_helper__/test-utils';
 import 'jest-styled-components';
```

Comment:
I don't think we need to `import 'jest-styled-components';` anymore on a per file basis as it has been added to the `jest.conf.json`. It may be that you need to update with master to surface that change though

Reply:
https://github.com/Sage/carbon/blob/b8a1596ea2ed0088054b29f611bef82ee7eabb9e/jest.conf.json#L9

https://github.com/Sage/carbon/blob/b8a1596ea2ed0088054b29f611bef82ee7eabb9e/src/__spec_helper__/expect.js#L1